hdfscontents/hdfsmanager.py

```diff
@@ -17,6 +17,12 @@
 import mimetypes
 import nbformat
 from traitlets import Instance, Integer, Unicode, default
+try:
+    from hops import xattr
+except ImportError:
+    pass
+
+from hops.exceptions import RestAPIError
```

Comment:
Can you also move this to being inside the try statement?

Reply:
Thanks. It is also unused import. My bad.